starters/deno-oak-denodb/deps.ts

```diff
@@ -3,3 +3,5 @@
  */
 
 export { Application, Router, Context } from 'https://deno.land/x/oak@v11.1.0/mod.ts';
+
+export { DataTypes, Database, Model, PostgresConnector } from 'https://deno.land/x/denodb/mod.ts';
```

Comment:
Should we lock this to the current version? 
`https://deno.land/x/denodb@v1.1.0/mod.ts`

Reply:
good call